README.md

```diff
@@ -1,92 +1,111 @@
-# OOP school library
+# oop-libray
 
-> In this project, I will start building my school library app. I'm practicing design patterns for building OOP applications.
-## Built With
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
```

Comment:
- [ ] You are yet to remove this as  requested by the previous reviewer, please kindly do so :+1: